ui/chatview.go

```diff
@@ -35,6 +35,7 @@ var (
 	urlRegex            = regexp.MustCompile(`<?(https?://)(.+?)(/.+?)?($|\s|\||>)`)
 	spoilerRegex        = regexp.MustCompile(`(?s)\|\|(.+?)\|\|`)
 	roleMentionRegex    = regexp.MustCompile(`<@&\d*>`)
+	badEmoji            = regexp.MustCompile(`<(:\w+:)\d+>`)
```

Comment:
The variable name should be changed to `customEmojiRegex`.

Is `\w+` really correct? I am not quite sure what characters are possible for custom emoji names. It'd be nice to confirm this. On top of that, make sure that the RegEx isn't greedy and captures too much. Unit tests would help here!